eessi_bot_job_manager.py

```diff
@@ -291,6 +299,61 @@ def process_new_job(self, new_job):
 
         return True
 
+    def process_running_jobs(self, running_job):
```

Comment:
please add a docstring (also to `determine_running_jobs`)

Reply:
docstring in `determine_running_jobs` is OK, but still missing for `process_running_jobs`

Reply:
Done.